File: Fibonacci.dart
import 'dart:io';

void main() {
  stdout.write("Enter a number: ");
  int? num = int.parse(stdin.readLineSync()!);

  for (var i = 0; i < num; i++) {
    print("Position " +
        (i + 1).toString() +
        ": " +
        (Fibonacci_Recursion(i).toString()));
  }
}

int Fibonacci_Recursion(int n) {
  if (n < 2) {
    return n;
  } else {
    return Fibonacci_Recursion(n - 1) + Fibonacci_Recursion(n - 2);
  }
}

File: Fibonacci.py
def Fibonacci_Recursion(n):
    if (n < 2):
        return n
    else:
        return (Fibonacci_Recursion(n-1) + Fibonacci_Recursion(n-2))


number = int(input("Enter a number: "))
for i in range(number):
    print(Fibonacci_Recursion(i))

